tests/counterpoll_test.py

```diff
@@ -54,6 +55,20 @@ def test_port_buffer_drop_interval_too_short(self):
         assert result.exit_code == 2
         assert expected in result.output
 
+    def test_pg_drop_interval(self):
+        runner = CliRunner()
+        result = runner.invoke(counterpoll.cli.commands["pg-drop"].commands["interval"], ["10000"])
+        print(result.output)
+        assert result.exit_code == 0
```

Comment:
Need to include the check if the interval field is changed to the new value in config db

Reply:
Done